packages/formation-react/src/components/Telephone/Telephone.mdx

```diff
@@ -82,29 +82,26 @@ import Telephone, { CONTACTS, PATTERNS } from '@department-of-veterans-affairs/f
     <Telephone contact={CONTACTS.VA_BENEFITS} />
   </p>
 
-  <strong>Adding a pattern</strong>
-  <p>
-    <Telephone
-      contact={CONTACTS.VA_311}
-      pattern={PATTERNS.OUTSIDE_US}
-    />
-  </p>
+{' '}
```

Comment:
Is this from some linter? Would you add a screenshot to make sure that this doesn't end up in the html? If it isn't necessary, removing it would be preferable.